### PR TITLE
COE-978: c8y-oee-cli oee-profiles create add more options to create a profile w/o the need to use a template

### PR DESCRIPTION
**Pull request by namnhatpham**

### CODE REVIEW

**Review by mvoigt-sag — APPROVED** [review]

File: templates/profile.jsonnet [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Simulator Hi Frequency Availability - 1",
+  "name": "Simulator Testing Script - 1",
   "amountUnit": "",
   "locationId": "Matrix",
   "subscriptions": [
```
Comment (mvoigt-sag) [inline templates/profile.jsonnet]: I guess one important parameter is missing: the deviceId which points to the machine for this profile and will be its parent device

File: api/profiles.yaml [hunk 2]
```diff
@@ -76,25 +83,110 @@ commands:
...
-        default: 80
       
       - name: quality
         type: integer
         property: oeeTargets.quality
         description: OEE target quality
-        default: 80
+
+    bodyTemplates:
+      - type: jsonnet
```
Comment (mvoigt-sag) [inline api/profiles.yaml:103]: how does this template definition relate to the "profile.jsonnet" file? is that just an example or is that really used at runtime?

File: api/profiles.yaml [hunk 3]
```diff
@@ -35,6 +35,13 @@ commands:
     description: Create profile
     method: POST
     path: service/oee-bundle/configurationmanager/2/configuration
+    exampleList:
+      - description: Syntax to create OEE profile
+        command: c8y oee profiles create --deviceId [device-id] --name [oee-profile-name]
```
Comment (mvoigt-sag) [inline api/profiles.yaml:40]: - will this really work without specifying a profile.jsonnet file? Does it use the bodyTemplates.template?
- if we keep this example, please change the description similar to the description of the following example

**Review by mvoigt-sag — CHANGES_REQUESTED** [review]

File: api/profiles.yaml (hunk 2 above)
Comment (namnhatpham) [reply api/profiles.yaml:103]: no, this is a template that is always applied to the body of the HTTP request and if users do not define any changes, the request body will be the same as in this bodyTemplates section.
This template doesn't have a relation to the template file

File: api/profiles.yaml (hunk 3 above)
Comment (namnhatpham) [reply api/profiles.yaml:40]: yes, as I mentioned, anything is defined in the bodyTemplates section will be set as default body of HTTP request if there are no further changes so the use of template is not necessary.
The syntax here will use the default body from bodyTemplates section and adding deviceId and name fields to it

**Review by mvoigt-sag — APPROVED** [review]

**Pull request merged**